src/mono/netcore/CoreFX.issues.rsp

```diff
@@ -5,86 +5,17 @@
 -notrait category=failing
 -notrait category=IgnoreForCI
 
-####################################################################
-##  Microsoft.VisualBasic.Core.Tests
-####################################################################
-
-# https://github.com/mono/mono/issues/14854
--nomethod Microsoft.VisualBasic.Tests.ErrObjectTests.Clear
-
-# Windows-only tests, incorrect platform guards (https://github.com/dotnet/corefx/pull/40365)
--nomethod Microsoft.VisualBasic.Tests.InteractionTests.SaveSetting
--nomethod Microsoft.VisualBasic.Tests.InteractionTests.DeleteSetting
--nomethod Microsoft.VisualBasic.Tests.InteractionTests.GetAllSettings
--nomethod Microsoft.VisualBasic.Tests.InteractionTests.GetSetting
-
-####################################################################
-##  System.Collections.Specialized.Tests
-####################################################################
-
-# TODO: Wrong exception
-# https://github.com/mono/mono/issues/14858
--nomethod System.Collections.Specialized.Tests.NameValueCollectionCtorTests.Ctor_NegativeCapacity_ThrowsArgumentOutOfRangeException
-
-####################################################################
-##  System.Collections.Concurrent.Tests
-####################################################################
-
-# fails only with release mono
--nomethod System.Collections.Concurrent.Tests.ConcurrentQueueTests.ReferenceTypes_NulledAfterDequeue
```

Comment:
the test was removed from the .rsp without adding a corresponding `[ActiveIssue]`, it causes one of the failures in your PR.

Reply:
Heh, it passed locally with Mono built in release configuration using `./build.sh --subsetCategory mono -configuration Release /p:__BuildType=Release`. Then I need to file an issue.

Reply:
I found related Mono issue but it's already closed https://github.com/mono/mono/issues/16413

Reply:
Reopened the issue for this test